src/task.rs

```diff
@@ -82,15 +82,14 @@ impl TaskMetrics {
         )
         .unwrap();
 
-        let first_poll_count = IntGaugeVec::new(
+        let first_poll_count = IntCounterVec::new(
```

Comment:
AFAIK, `first_poll_count` represents the number of tasks that were polled for the first time within a given sampling **interval**, so it can fluctuate up and down. https://docs.rs/tokio-metrics/latest/tokio_metrics/struct.TaskMetrics.html#structfield.first_poll_count

Perhaps we should add a `total_first_poll_count`.

Reply:
Oh thats true, let me clean that up ;)